include/oboe/FullDuplexStream.h

```diff
@@ -24,26 +24,68 @@
 
 namespace oboe {
 
-class FullDuplexStream : public AudioStreamCallback {
+/**
+ * FullDuplexStream can be used to synchronize an input and output stream.
+ *
+ * For the builder of the output stream, call setDataCallback() with this stream.
+ *
+ * When both streams are ready, onAudioReady() will call onBothStreamsReady().
+ * Callers must override onBothStreamsReady().
+ *
+ * To ensure best results, open an output stream before the input stream.
+ * Call inputBuilder.setBufferCapacityInFrames(mOutputStream->getBufferSizeInFrames() * 2).
```

Comment:
The new test uses Capacity instead of Size:
    mInputBuilder.setBufferCapacityInFrames(mOutputStream->getBufferCapacityInFrames() * 2);

I think that is better.